index.html

```diff
@@ -19,20 +21,31 @@
 		</script>
 	</head>
 	<body>
-		<main class="container">
-			
 			<header class="portfo-header">
-				<a href="#" class="mylogo"> My Logo</a>
-				<nav class="menu-button" onclick="toggle()">
-					<ul class="desk-nav">
+				<div class="mylogo"><h1>Beyk</h1>
+				</div>
+				<nav class="nav">
+				<div class="mobile-nav">
+					<div class="openMenu">
+						<i class="fa fa-bars"></i>
+					</div>
+					<ul class="mainMenu">
+						<li><a href="#work" class="menu-Links">portfolio</a></li>
+							<li><a href="#about-me" class="menu-Links">about</a></li>
+							<li><a href="#contact-me" class="menu-Links">contact</a></li>
```

Comment:
- [ ] I think it would be better if you `capitalize the first letters of the links and make the `font sizes larger` to stick more with the original design.
- Original design:
![Screenshot (172)](https://user-images.githubusercontent.com/67087664/152298617-798e3833-8d66-4611-9674-83c4981d3823.png)
- Your design is below:
![Screenshot (173)](https://user-images.githubusercontent.com/67087664/152298742-471e5287-0669-44b6-ac17-5493dbc34e63.png)
Kindly fix it 🙏